### PR TITLE
Added notification bar to store and homepage

### PR DESCRIPTION
**Pull request by tbille**
see https://github.com/canonical-websites/snapcraft.io/pull/1564

### CODE REVIEW

File: templates/partials/livestream-notification.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <a href="https://www.youtube.com/watch?v=BEp_l2oUcD8&amp;ab_channel=snapcraft" target="_blank" class="p-link--external">Watch our Youtube Live Stream on building snaps</a> this Wednesday 6 Feb 2019 @ 20:00 GMT
+        </p>
+    </div>
+</div>
+{% elif is_stream_live == 1 %}
+<div class="col-12">
+    <div class="p-notification--information p-snapcraft-nps-banner">
+        <p class="p-notification__response">
+        <span class="p-notification__status">LIVE:</span>
+        <a href="https://www.youtube.com/watch?v=BEp_l2oUcD8&amp;ab_channel=snapcraft" target="_blank" class="p-link--external">Watch our Youtube Live Stream on building snaps</a> is now!
```
Comment (bartaz) [inline templates/partials/livestream-notification.html]: "Watch our Youtube Live Stream on building snaps is now!" doesn't sound right. I guess we could drop "is"?

Comment (tbille) [reply templates/partials/livestream-notification.html]: lol I didnt' read

Comment (tbille) [reply templates/partials/livestream-notification.html]: better like this?
image

Comment (therealjuan) [reply templates/partials/livestream-notification.html]: 👍

**Pull request merged**